toolchain/lower/file_context.h

```diff
@@ -188,12 +196,20 @@ class FileContext {
 
   // Build the DISubprogram metadata for the given function.
   auto BuildDISubprogram(const SemIR::Function& function,
+                         SemIR::SpecificId specific_id,
                          const llvm::Function* llvm_function)
       -> llvm::DISubprogram*;
 
+  auto BuildDISubroutineType(
+      const SemIR::Function&, SemIR::SpecificId specific_id,
+      llvm::SmallVectorImpl<llvm::DIType*>& debug_parameter_types) const
+      -> llvm::DISubroutineType*;
+  auto BuildDIType(SemIR::TypeId type_id) const -> llvm::DIType*;
```

Comment:
Add comments for these?

Reply:
Done in a9c8977f9ae7ca2bf3623bfb470d390925f25da1 (the `BuildDIType` was a leftover/dead declaration, so removed that)